.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: objective-c
-xcode_project: VelocidiSDK.xcodeproj # path to your xcodeproj folder
+os: osx
+xcode_project: VelocidiSDK/VelocidiSDK.xcodeproj # path to your xcodeproj folder
 xcode_scheme: VelocidiSDKTests
-xcode_destination: platform=iOS Simulator,OS=10.1,name=iPad Pro (9.7-inch)
+xcode_destination: platform=iOS Simulator,OS=11.4,name=iPhone X
```

Comment:
Is it worth to test other iOS versions?

The most recent version is iOS 12 is [installed on 78% of all devices from the last four years (as of January 2019)](https://www.macrumors.com/2019/01/02/ios-12-installed-on-78-percent-of-devices/). The following widest version is iOS 11. Also, while [users on iOS 11 get a chance to update, users on iOS 10 don't](https://everyi.com/by-capability/maximum-supported-ios-version-for-ipod-iphone-ipad.html). The same logic applies to the rest of the iOS versions.

My take is that, depending on the minimum iOS version supported by the SDK, we should test the major versions starting from the minimum. We can take advantage of the [build matrix](https://docs.travis-ci.com/user/build-matrix/) to test the different versions.

What do you think?